app/views/pages/services_using_registers.html.haml

```diff
@@ -17,82 +15,82 @@
           %tr
             %th E-petitions
             %td Parliamentary Digital Service
-            %td= link_to 'Country', 'https://country.register.gov.uk', target: :blank
+            %td= link_to 'Country', '/registers/country'
           %tr
             %th GOV.UK Pay
             %td Cabinet Office
             %td
-              = link_to 'Country', 'https://country.register.gov.uk', target: :blank
+              = link_to 'Country', '/registers/country'
```

Comment:
can we resolve these using the router paths?

Reply:
Maybe but im not sure its worth it...